extensions-core/multi-stage-query/pom.xml

```diff
@@ -352,6 +352,17 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Yeah, the MSQ query benchmark implementation is relying on the MSQ module's test classes. This change is needed so that the test JAR is generated, for use of the benchmark module.